File: TabNine/JavaScript/Q7.js
/**
 * @param {number} n
 * @return {number}
 */
var countPrimes = function(n) {
    //Given an integer n, return the number of prime numbers that are strictly less than n.
    //constraint: 0 <= n <= 5 * 10^6


    if(n <= 1) return 0;

    let count = 0;

    for(let i = 2; i < n; i++){

        if(isPrime(i)){
            count++;
        }
    }

    return count

    function isPrime(n) {
        if(n <= 1) return false;

        for(let i = 2; i < n; i++){
            if(n % i === 0) return false;
        }

        return true;
    }
}

File: TabNine/Java/Q7.java
package Java;

public class Q7 {
    public int countPrimes(int n) {
        //Given an integer n, return the number of prime numbers 
        //that are strictly less than n. with 0 <= n <= 5 * 106
    
        int count = 0;
        for (int i = 2; i < n; i++) {
            if (isPrime(i)) {
                count++;
            }
        }
        return count;
    
}

    private boolean isPrime(int i) {
        return false;
    }
}
